src/DDTrace/Integrations/Yii/V2/YiiIntegrationLoader.php

```diff
@@ -0,0 +1,114 @@
+<?php
+
+namespace DDTrace\Integrations\Yii\V2;
+
+use DDTrace\Configuration;
+use DDTrace\Contracts\Scope;
+use DDTrace\GlobalTracer;
+use DDTrace\Http\Urls;
+use DDTrace\Integrations\Yii\YiiSandboxedIntegration;
+use DDTrace\Integrations\Integration;
+use DDTrace\SpanData;
+use DDTrace\Tag;
+use DDTrace\Type;
+use yii\helpers\Url;
+
+class YiiIntegrationLoader
+{
+
+    public function load(YiiSandboxedIntegration $integration)
+    {
+        $scope = GlobalTracer::get()->getRootScope();
+        if (!$scope instanceof Scope) {
+            return Integration::NOT_LOADED;
+        }
+        $root = $scope->getSpan();
+        // Overwrite the default web integration
+        $root->setIntegration($integration);
+        $root->setTraceAnalyticsCandidate();
+        $root->overwriteOperationName('yii.request');
```

Comment:
I think that operation name should still be `web.request`, here and for the future integrations. The reason why I am saying this is based on the following scenario.

We do not support a framework, so given the url to resource name will be `name: web.request, resource: GET /some/path`. When we start to support a framework it becomes `name: yii.request, resource: GET /some/path`. I don't see a benefit for this. What about adding a tag `app.framework: yii` to the root span and leave `web.request`?

Reply:
There's already `integration.name` set on it, with the value of `yii`. I guess we could add `app.framework.name=yii` and `app.framework.version=\Yii::getVersion()` if you think customers will think that's helpful. Otherwise I'm fine with `integration.name`.

Reply:
> I guess we could add app.framework.name=yii and app.framework.version=\Yii::getVersion() if you think customers will think that's helpful
I did not remember of `integration.name`. So forget about it.

What do you think about `web.request` instead?

Reply:
I'm fine with `web.request` instead of `yii.request`. @SammyK , what do you think?

Reply:
I think @labbati makes a good point. I'm cool with keeping it `web.request` :)